packages/asset-kit/src/components/InformationSection.tsx

```diff
@@ -0,0 +1,73 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { hasRichTextValue, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
+import {
+    BoldPlugin,
+    ItalicPlugin,
+    PluginComposer,
+    ResetFormattingPlugin,
+    RichTextEditor,
+    SoftBreakPlugin,
+    StrikethroughPlugin,
+    TextStylePlugin,
+    UnderlinePlugin,
+} from '@frontify/fondue';
+import { Plugin, PluginProps } from '@frontify/fondue/dist/components/RichTextEditor/Plugins/Plugin';
+import { InformationSectionProps } from '../types';
+import { useMemo } from 'react';
+
+const DEFAULT_CONTENT_VALUE = '[{"type":"heading3","children":[{"text":""}]}]';
+
+export const InformationSection = ({ description, isEditing, setBlockSettings, title }: InformationSectionProps) => {
+    const { designTokens } = useGuidelineDesignTokens();
+
+    const customTitlePlugins = useMemo(() => {
+        return new PluginComposer()
+            .setPlugin([new SoftBreakPlugin(), new TextStylePlugin() as Plugin<PluginProps>])
+            .setPlugin([new BoldPlugin(), new ItalicPlugin(), new UnderlinePlugin(), new StrikethroughPlugin()])
+            .setPlugin([new ResetFormattingPlugin()]);
+    }, []);
+
+    const saveDescription = (newDescription: string) => {
+        if (description !== newDescription) {
+            setBlockSettings({ description: newDescription });
+        }
+    };
+
+    const saveTitle = (newTitle: string) => {
+        if (title !== newTitle) {
+            setBlockSettings({ title: newTitle });
+        }
+    };
+
+    return (
+        <div className="tw-flex-1 tw-space-y-2">
+            {hasRichTextValue(title) || isEditing ? (
+                <div data-test-id="block-title">
+                    <RichTextEditor
+                        designTokens={designTokens}
+                        value={title ?? DEFAULT_CONTENT_VALUE}
+                        readonly={!isEditing}
+                        onBlur={saveTitle}
+                        placeholder={isEditing ? 'Add a title here ...' : ''}
+                        border={false}
+                        plugins={customTitlePlugins}
+                    />
+                </div>
+            ) : null}
```

Comment:
You can use the html serializer for the view mode. This way we would render the RTE only in edit mode. You can check how it's done in the text block, but it would be something like this:

```suggestion
            <div data-test-id="block-title">
                {!isEditing ? (
                    <div data-test-id="rte-content-html" dangerouslySetInnerHTML={{ __html: html }} />
                ) : (
                    <RichTextEditor
                        designTokens={designTokens}
                        value={title ?? DEFAULT_CONTENT_VALUE}
                        readonly={!isEditing}
                        onBlur={saveTitle}
                        placeholder={isEditing ? 'Add a title here ...' : ''}
                        border={false}
                        plugins={customTitlePlugins}
                    />
                )}
            </div>
```

and this:
```
    const rawValue = JSON.stringify(parseRawValue({ raw: blockSettings.content ?? '' }));
    const html = serializeRawToHtml(rawValue, designTokens, blockSettings.columnNumber, gap);
```

Reply:
adjusted 👍